Package.swift

```diff
@@ -0,0 +1,44 @@
+// swift-tools-version:5.9
+import PackageDescription
+
+let package = Package(
+    name: "TreeSitterPkl",
+    products: [
+        .library(name: "TreeSitterPkl", targets: ["TreeSitterPkl"]),
+    ],
+    dependencies: [],
+    targets: [
+        .target(name: "TreeSitterPkl",
+                path: ".",
+                exclude: [
+                    "binding.gyp",
+                    "bindings",
+                    "Cargo.lock",
+                    "Cargo.toml",
+                    "corpus",
+                    "grammar.js",
+                    "LICENSE.txt",
+                    "NOTICE.txt",
+                    "package.json",
+                    "package-lock.json",
+                    "tsconfig.json",
+                    "README.adoc",
+                    "SECURITY.adoc",
+                    "MAINTAINERS.adoc",
+                    "CODE_OF_CONDUCT.adoc",
+                    "CONTRIBUTING.adoc",
+                    "src/grammar.json",
+                    "src/node-types.json"
+                ],
+                sources: [
+                    "src/parser.c",
+                    "src/scanner.c",
+                ],
+                resources: [
+                    .copy("queries"),
+                ],
+                publicHeadersPath: "bindings/swift",
+                cSettings: [.headerSearchPath("src")]
+        )
+    ]
```

Comment:
I like the idea of providing a more SPM compatible interface, but this smells lie it's bound to (quietly!) go out of sync. Shouldn't this be generated instead? At least, is there something we can do to reliably test this in CI (see `.circleci/config.pkl`)

Reply:
Apologies for jumping in here, I just thought this might be useful information. I have added SPM support to ~ 30 tree-sitter parsers. So far, 1 has gotten stale. Only affected Swift users that weren't pinning to a specific version, and was easy to fix.

Also, using manual `exclude` like this was to silence warnings from earlier SPM versions. I'm pretty sure that this is no longer necessary and you can just omit this part. And now that C++ scanners are deprecated, I have a feeling that as long as you continue to check in the generated `parser.c` things will work without issue.

Reply:
There's never a need to apologise for constructive contributions, if you ask me ;) Thanks for the input.

Given `// swift-tools-version:5.9` (is that a hard failure when on an older (newer?) version?) and a general tendency to be relatively up to date in the ecosystem, I'm happy to take this. @jayadamsmorgan, do you want to have peek at whether the excludes can be omitted?

Reply:
There shouldn't be any issues with the newer versions, older versions just seem to give some warnings
I am trying omitting `exclude` right now.

Reply:
Confirming no issues after omitting